governance.md

```diff
@@ -1,149 +1,285 @@
-  write access to repo (assign issues/PRs, add/remove labels and milestones, edit issues and PRs, edit wiki,
-  create/delete labels and milestones); technically can lgtm any PR and cause it
-  to be merged by the submit queue, but expected to respect OWNERS files; expected to review PRs, fix bugs, maintain and
-  improve health and quality of the project, provide user support, mentor and guide approvers,
-  reviewers, and other contributors; must apply to `kubernetes-maintainers@googlegroups.com`, with a
-  [Champion](https://github.com/kubernetes/community/blob/master/incubator.md#faq) from the existing
-  kubernetes-maintainers members and a Sponsor from Project Approvers, with a summary
-  of contributions to the project, current project responsibilities, and links to merged and assigned PRs;
-  at least 3 of the maintainers must approve the application, with no objections; the application
-  expires after 2 weeks if not enough approvals are granted
-- **Project Approvers**: approver in [top-level OWNERS file in kubernetes repo](https://github.com/kubernetes/kubernetes/blob/master/OWNERS);
-  de-facto project decision makers; technically can 
-  approve virtually any PRs; can sponsor incubator repos; can sponsor maintainers;
-  maintainer in good standing for at least 1 year; strong technical vision;
-  committed to project's mission and culture; nomination/application process TBD
-- [**API Approver**](https://github.com/orgs/kubernetes/teams/api-approvers):
-  lead designers of the project, who are familiar with the 
-  design, requirements, mechanics, conventions, style, scope, gotchas, etc. 
-  of the API; most beta/GA API changes are vetted by the API approvers
-- [**API Reviewer**](https://github.com/orgs/kubernetes/teams/api-reviewers):
-  contributors familiar with design, requirements, mechanics, conventions, style,
-  scope, gotchas, etc. of the API; have written and/or reviewed Kubernetes APIs
+document, familiarity with project organization, conventions, and policies & procedures, as well as technical and/or writing abilities. Role-specific expectations, responsibilities, and requirements
+are enumerated below. Because github teams are not visible by nonmembers of the org and because changes
+to them are not transparent, the primary documentation of role membership beyond MEMBER should be 
+maintained in OWNERS files in the repository.
+
+- **MEMBER**:
+  - Requirements
+    - an active contributor for at least 3 months
+    - authored and/or reviewed at least 10 merged non-trivial PRs
+      - TBD: figure out how to count reviews
+    - active enough to be assigned issues and/or PRs, and to be added to a github team
+      (a SIG, for example) for notification purposes
+    - has enabled [GitHub’s two-factor authentication](https://help.github.com/articles/about-two-factor-authentication/)
+    - nominated by a REVIEWER, APPROVER, MAINTAINER, OWNER, or LEAD on kubernetes-dev@googlegroups.com,
```

Comment:
@cblecker How one gets nominated hasn't been figured out yet. I've been asking for someone to build a contributor metrics dashboard so that we could track candidates.
https://github.com/kubernetes/contrib/issues/1029